code/game/mecha/working/ripley.dm

```diff
@@ -180,4 +180,13 @@
 	else
 		step_in = 5
 		for(var/obj/item/mecha_parts/mecha_equipment/drill/drill in equipment)
-			drill.equip_cooldown = initial(drill.equip_cooldown)
\ No newline at end of file
+			drill.equip_cooldown = initial(drill.equip_cooldown)
+
+/obj/mecha/working/ripley/emag_act(mob/user)
+	if(!emagged)
+		emagged = TRUE
+		to_chat(user, "<span class='notice'>You slide the card through [src]'s ID slot.</span>")
+		playsound(loc, "sparks", 100, 1)
+		desc += "</br><span class='danger'>The mech's equipment slots spark dangerously!</span>"

```

Comment:
As an additive description, trying to describe something we should be physically seeing to fall in line with the rest of the codebase seems goofy. If the mech was actively sparking on a regular basis then fair enough, but seems over the top.

Reply:
Same here. Old part of the codebase.